sandy/njoy.py

```diff
@@ -670,6 +670,10 @@ def _errorr_input(endfin, pendfin, errorrout, mat,
     2 /
     1.00000e-02 1.00000e+03 2.00000e+05 /
     """
+    if ek is not None:
+        ign = 1
+    else:
```

Comment:
this exception will never happen. check the tests! Maybe you can remove it

Reply:
I will remove the test that fails.